src/subcmd/withdraw_fees.rs

```diff
@@ -0,0 +1,71 @@
+use base64::{engine::general_purpose, Engine};
+use bincode::serialize;
+use clap::Args;
+use solana_program::{program_pack::Pack, system_instruction::withdraw_nonce_account};
+use solana_sdk::{message::Message, pubkey::Pubkey, transaction::Transaction};
+use stakedex_sdk_common::find_fee_token_acc;
+
+use super::SubcmdExec;
+
+#[derive(Args, Debug)]
+#[command(
+    long_about = "Outputs a base64-encoded transaction that withdraws fees to the specified token account when signed by the admin authority."
+)]
+pub struct WithdrawFeesArgs {
+    #[arg(help = "Mint of the token to withdraw fees for.")]
+    pub mint: Pubkey,
+    #[arg(
+        help = "Destination token account to withdraw fees to. Defaults to the admin authority's associated token account if not provided."
+    )]
+    pub destination: Option<Pubkey>,
+}
+
+const ADMIN_AUTHORITY: &str = "A7jn1BA6LPHX8Wcmc8t476gjoLCG4PZakww19ZXfFRjX";
+
+impl SubcmdExec for WithdrawFeesArgs {
+    fn process_cmd(&self, args: &crate::Args) {
+        let client = args.config.rpc_client();
+        let payer = args.config.signer();
+
+        let mint_token_account = find_fee_token_acc(&self.mint).0;
+        let destination_token_account = find_fee_token_acc(
+            &self
+                .destination
+                .unwrap_or_else(|| ADMIN_AUTHORITY.parse().unwrap()),
+        )
+        .0;
```

Comment:
`destination` should be a token account, not a wallet/systemAccount

```suggestion
        let destination_token_account = destination.unwrap_or_else(|| find_fee_token_acc(ADMIN_AUTHORITY.parse().unwrap()).0);
```